pkg/apiroutes/templates.go

```diff
@@ -166,17 +184,22 @@ func AddTemplateRepo(URL, description string, name string) ([]utils.TemplateRepo
 
 // DeleteTemplateRepo deletes a template repo from PFE and
 // returns the new list of existing repos
-func DeleteTemplateRepo(URL string) ([]utils.TemplateRepo, error) {
+func DeleteTemplateRepo(URL string, conID string) ([]utils.TemplateRepo, error) {
```

Comment:
Same as previous comment on setting data types when they are all the same

Reply:
done